File: VB/AnimationExample/MainForm.vb
Imports DevExpress.XtraCharts
Imports System

Namespace AnimationExample

    Public Partial Class MainForm
        Inherits DevExpress.XtraEditors.XtraForm

        Private ReadOnly Property series As Series
            Get
                Return chartControl.Series(0)
            End Get
        End Property

        Private ReadOnly Property diagram As XYDiagram
            Get
                Return TryCast(chartControl.Diagram, XYDiagram)
            End Get
        End Property

        Private ReadOnly Property view As LineSeriesView
            Get
                Return TryCast(series.View, LineSeriesView)
            End Get
        End Property

        Public Sub New()
            InitializeComponent()
        End Sub

#Region "#AnimationConfig"
        Protected Overrides Sub OnLoad(ByVal e As EventArgs)
            MyBase.OnLoad(e)
            chartControl.AnimationStartMode = ChartAnimationMode.OnLoad
            view.SeriesPointAnimation = New FanMarkerAnimation() With {.BeginTime = New TimeSpan(0, 0, 0), .Duration = New TimeSpan(0, 0, 0, 1, 500), .PointDelay = New TimeSpan(0, 0, 0, 0, 250), .PointOrder = PointAnimationOrder.Random, .EasingFunction = New QuinticEasingFunction With {.EasingMode = EasingMode.Out}}
        End Sub

        Private Sub OnAnimateClick(ByVal sender As Object, ByVal e As EventArgs)
            chartControl.Animate()
        End Sub
#End Region  ' #AnimationConfig
    End Class
End Namespace

File: CS/AnimationExample/MainForm.cs
using DevExpress.XtraCharts;
using System;

namespace AnimationExample {
    public partial class MainForm : DevExpress.XtraEditors.XtraForm {
        Series series { get { return chartControl.Series[0]; } }
        XYDiagram diagram { get { return chartControl.Diagram as XYDiagram; } }
        LineSeriesView view { get { return series.View as LineSeriesView; } }

        public MainForm() {
            InitializeComponent();
        }

        #region #AnimationConfig
        protected override void OnLoad(EventArgs e) {
            base.OnLoad(e);

            chartControl.AnimationStartMode = ChartAnimationMode.OnLoad;
            view.SeriesPointAnimation = new FanMarkerAnimation() {
                BeginTime = new TimeSpan(0, 0, 0),
                Duration = new TimeSpan(0, 0, 0, 1, 500),
                PointDelay = new TimeSpan(0, 0, 0, 0, 250),
                PointOrder = PointAnimationOrder.Random,
                EasingFunction = new QuinticEasingFunction {
                    EasingMode = EasingMode.Out
                }
            };
        }

        private void OnAnimateClick(object sender, EventArgs e) {
            chartControl.Animate();
        }
        #endregion #AnimationConfig
    }
}

